src/basic/terminal-util.h

```diff
@@ -11,19 +11,35 @@
 #include "time-util.h"
 
 /* Regular colors */
+#define ANSI_BLACK   "\x1B[0;30m"
 #define ANSI_RED     "\x1B[0;31m"
 #define ANSI_GREEN   "\x1B[0;32m"
 #define ANSI_YELLOW  "\x1B[0;33m"
 #define ANSI_BLUE    "\x1B[0;34m"
 #define ANSI_MAGENTA "\x1B[0;35m"
+#define ANSI_CYAN    "\x1B[0;36m"
+#define ANSI_WHITE   "\x1B[0;37m"
+
+#define ANSI_BRIGHT_BLACK   "\x1B[0;90m"
+#define ANSI_BRIGHT_RED     "\x1B[0;91m"
+#define ANSI_BRIGHT_GREEN   "\x1B[0;92m"
+#define ANSI_BRIGHT_YELLOW  "\x1B[0;93m"
+#define ANSI_BRIGHT_BLUE    "\x1B[0;94m"
+#define ANSI_BRIGHT_MAGENTA "\x1B[0;95m"
+#define ANSI_BRIGHT_CYAN    "\x1B[0;96m"
+#define ANSI_BRIGHT_WHITE   "\x1B[0;97m"
+
 #define ANSI_GREY    "\x1B[0;38;5;245m"
 
 /* Bold/highlighted */
+#define ANSI_HIGHLIGHT_BLACK             "\x1B[0;1;30m"
```

Comment:
this is supposed to be some form of grey, iiuc, should hence have a comment about this